vcr/interface.go

```diff
@@ -68,34 +69,38 @@ type Writer interface {
 	StoreRevocation(r credential.Revocation) error
 }
 
+// Resolver binds all read type of operations into an interface
+type Resolver interface {
+	// Registry returns the concept registry
+	Registry() concept.Registry
```

Comment:
Maybe better to have it expose (and delegate) `QueryFor`? Because now you have access to `ConceptTemplate`, `Add`, `Transform` which looks a bit strange to me given its a resolving (read-only).

Reply:
I split the Registry interface int to 2 parts. Also transform doesn't transform anything internal, it transforms the given object.